packages/web3-types/src/web3_api_types.ts

```diff
@@ -31,6 +31,7 @@ export interface EthSubscription extends ProviderMessage {
 }
 
 export interface ProviderRpcError extends Error {
+	message: string;
```

Comment:
The Error interface provided by TypeScript already contains a `message` (and also a `name`) 
```suggestion
```

Here is the definision of the Error interface:
```ts
interface Error {
    name: string;
    message: string;
    stack?: string;
}
```